pkgs/development/libraries/libieee1284/default.nix

```diff
@@ -24,7 +34,7 @@ stdenv.mkDerivation rec {
     "--without-python"
   ];
 
-  preConfigure = ''
+  prePatch = ''
```

Comment:
Should this be postPatch?

Reply:
No, because the patch applies to files that are generated by the bootstrap script.